jscomp/super_errors/super_misc.mli

```diff
@@ -0,0 +1,6 @@
+(** Range coordinates all 1-indexed, like for editors. Otherwise this code
+  would have way too many off-by-one errors *)
+val print_file: range:(int * int) * (int * int) -> lines:string array -> Format.formatter -> unit -> unit
+
+(** This will be called in super_main. This is how you override the default error and warning printers *)
+val colorize_tagged_string: Format.formatter -> (string -> Ext_color.style list) -> unit
```

Comment:
I've tried just adding tags to ext_colors but then I realized it's not actually called. The existing colors are called by bsb_main's `Bsb_log.setup ()`, and maybe at that point it's already too late, the ninja command has exited (correct me if I'm wrong).

So I'll end up with a `Super_misc.setup`, or the current `colorize_tagged_string` anyway, and call it at the appropriate time. But if you'd like to avoid the complexity of adding the option arg in ext_colors, I can copy it over to a super_* file?